apps/cowswap-frontend/src/legacy/components/Header/Polling.tsx

```diff
@@ -157,25 +154,18 @@ const Wrapper = styled.div`
   }
 `
 
-const DEFAULT_MS_BEFORE_WARNING = ms`10m`
-const NETWORK_HEALTH_CHECK_MS = ms`10s`
-
 export function Polling() {
```

Comment:
not from this PR, but this is not a great name for this component

It looks like is just sth to show the blocknumber and gas price, so `Polling`` is not super good

Reply:
Yeah, agree, we should rename it. I would do this in the next PR to not blow this one